app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageExpiration.kt

```diff
@@ -218,21 +220,25 @@ class SelfDeletionTimer(private val context: Context) {
              * alpha value is equal to the ratio = 1 / 2.5 = 0.4
              */
             fun alphaBackgroundColor(): Float {
-                val timeElapsed = expireAfter - timeLeft
-                val timeElapsedRatio = timeElapsed / expireAfter
+                return if (timeLeft != ZERO) {
```

Comment:
Could there be any possibility that `timeLeft` is lower than `0`? if so or maybe just to be on the safe side we could change `!=` to `timeLeft > ZERO`?

Reply:
Not really possible as we normalize it on Kalium side, but worth adding it thanks <3